src/routes/tickets/controllers/getUserAuthoredTickets.ts

```diff
@@ -15,7 +15,17 @@ const getUserAuthoredTickets = async (req, res) => {
 
   const tickets = await find(getDB().tickets, {
     limit: parseInt(process.env.PAGINATION_LIMIT, 10),
-    query: { authorId: req.query.userId, status: TicketStatus.open },
+    query: {
+      authorId: req.params.userId,
```

Comment:
Do you have input validation?

Reply:
Very simple check if the userId exists. I should add better validation here.

Reply:
Fixed.